### ISSUE TITLE
Allow passing query params to GET method directly

### ISSUE THREAD

**Issue by KaKi87** [axios/axios#6150]
### Is your feature request related to a problem? Please describe.

The `axios.post()` request method alias is indeed convenient but the `axios.get()` one actually isn't as it requires the same things as `axios()`.

### Describe the solution you'd like

Just like there's `axios.post(url[, data[, config]])`, there could be `axios.get(url[, params[, config]])`.

### Describe alternatives you've considered

None

### Additional context/Screenshots

The same convenience could also be added to DELETE/HEAD/OPTIONS like POSTS's is also added to PUT/PATCH.

---
Thanks

**Comment by jasonsaayman** [comment]
i dont think the method signature is something we would be chaning soon, but i could welcome a non-breaking PR, however for now i am going to close this issue

**Comment by KaKi87** [comment]
> i dont think the method signature is something we would be chaning soon

What about for a v2 ?

> non-breaking

What would that be ?

**Comment by jasonsaayman** [comment]
yeah fair i guess there is no "non-breaking" manner to do this... i will add as someting to look at in v2
